### PR TITLE
Add mention of Google Colab option

### PR DESCRIPTION
**Pull request by bertiewooster**
Seems like the Google Colab option was added, then not mentioned as an option on this page. I found logging in with a Microsoft account extremely finicky, whereas Google Colab requires no account or login, so I recommend listing Google Colab as an option.

### CODE REVIEW

**Comment by svekars** [comment]
Thank you for your contribution. Before we can review your PR, please sign the CLA.

**Comment by bertiewooster** [comment]
@svekars hmm, not sure what's going wrong--I've signed the CLA twice now, but it's not getting noted:

meta-CLA-done

File: beginner_source/basics/intro.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 ------------------
 You can run this tutorial in a couple of ways:
 
-- **In the cloud**: This is the easiest way to get started! Each section has a "Run in Microsoft Learn" link at the top, which opens an integrated notebook in Microsoft Learn with the code in a fully-hosted environment.
+- **In the cloud**: This is the easiest way to get started! Each section has a "Run in Microsoft Learn" and "Run in Google Colab" link at the top, which opens an integrated notebook in Microsoft Learn (Microsoft account required) or Google Colab (no account required), respectively, with the code in a fully-hosted environment.
```
Comment (svekars) [inline beginner_source/basics/intro.py:37]: LGTM, but can we remove the part about accounts please? This is something Microsoft and Google control and can change any time and we might miss.

Comment (bertiewooster) [reply beginner_source/basics/intro.py]: Done. Needing a Microsoft account (and logging into it) was a big hassle for me, so I was trying to give learners a hint about that, but I appreciate your point.

**Review by svekars — APPROVED** [review]
Thanks! LGTM!

**Pull request merged**